### PR TITLE
Added readme and basic model tests

### PR DESCRIPTION
**Pull request by mish24**
This PR has the following changes:
1. Documentation and requirements added.
2. Unit tests using pytest for Pcsa models.
3. Functional tests for dashboard.

### CODE REVIEW

File: MACCTests2017/tests/unit/test_models_pcsa.py [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+setting up a user to test the data with
+"""
+class UserTest(TestCase):
+
+	def setUp(self):
+		self.username = "testuser"
+		self.email = "testuser@gmail.com"
+		self.password = "password"
+
+		self.test_user = User.objects.create_user(username=self.username,
```
Comment (prachi1210) [inline MACCTests2017/tests/unit/test_models_pcsa.py]: Follow PEP8 style guidelines

File: MACCTests2017/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+### Description
+This repository contains the unit and functional tests written for the backend and frontend of the repository [MACC](https://github.com/systers/macc.git). The work is done for GSoC 2017 and has the changes made along with the updates in the repository.
+
+### Libraries used
+For unit testing of the Django app, I have mostly made use of `pytest`, `pytest-django` and the `unittest` frameworks.
+
+For functional testing of the app, the `Selenium` framework is used along with the help of pytest.
+
+To run the tests using the repository, please refer to the [requirements](https://github.com/mish24/automated-testing.git/MACCTests2017/requirements.txt) file, and copy the contents from there to the main [requirements.txt](https://github.com/systers/macc.git/requirements.txt) folder, or download them separately using `pip -m install library_name`.
```
Comment (prachi1210) [inline MACCTests2017/README.md]: Reference systers repo instead of personal repo

**Comment by prachi1210** [comment]
@mish24 please maintain a consistent styling throughout your code to maintain code quality. @khushbuparakh

**Comment by mish24** [comment]
@prachi1210 I added the PEP8 styling and all the changes.

File: MACCTests2017/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+Mobile App Control Center - Testing
+===================================
+
+### Description
+This repository contains the unit and functional tests written for the backend and frontend of the repository [MACC](https://github.com/systers/macc.git). The work is done for GSoC 2017 and has the changes made along with the updates in the repository.
+
+### Libraries used
+For unit testing of the Django app, I have mostly made use of `pytest`, `pytest-django` and the `unittest` frameworks.
```
Comment (prachi1210) [inline MACCTests2017/README.md]: @mish24 can you amend your commit to reflect this? then, we can merge this PR

Comment (mish24) [inline MACCTests2017/README.md]: @prachi1210 Done!

Comment (prachi1210) [inline MACCTests2017/README.md]: @mish24 are you sure you pushed your changes?

**Comment by prachi1210** [comment]
@mish24 for the future, please use separate commits only for changes that fix some bug or add a new feature. 
For changes reflecting code reviews, you can use `git commit --amend` followed by `git push -ff <remotename> <branchname>`

**Pull request merged**